test/e2e/framework/.import-restrictions

```diff
@@ -130,6 +130,40 @@
 			],
 			"ForbiddenPrefixes": []
 		},
+		{
+			"SelectorRegexp": "k8s[.]io/kubernetes/test/e2e/framework/",
+			"AllowedPrefixes": [],
+			"ForbiddenPrefixes": [
+				"k8s.io/kubernetes/test/e2e/framework/auth",

```

Comment:
Do we need this?  Wouldn't it by definition cause an import cycle?  
All subpackages would import framework.

Reply:
Sorry for my late response.

and you are right, all subpackages should be able to import the framework.
After understanding this verify-import-boss check way, I saw the above proposed change was invalid.
I will try to find the valid way to block circular import with this.